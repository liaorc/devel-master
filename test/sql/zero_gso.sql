--#
--#       Gpu Sort on Zero-record Table TestCases. 
--#

set pg_strom.debug_force_gpusort to on;
set pg_strom.gpu_setup_cost=0;
set random_page_cost=1000000;   --# force off index_scan.   
set pg_strom.enable_gpusort to on;                                                                                                                                       
set client_min_messages to warning;

select * from strom_zero_test order by smlint_x;
select * from strom_zero_test order by integer_x;
select * from strom_zero_test order by bigint_x;
select * from strom_zero_test order by real_x;
select * from strom_zero_test order by float_x;
select * from strom_zero_test order by nume_x;
