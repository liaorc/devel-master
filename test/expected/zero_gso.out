--#
--#       Gpu Sort on Zero-record Table TestCases. 
--#
set pg_strom.debug_force_gpusort to on;
set pg_strom.gpu_setup_cost=0;
set random_page_cost=1000000;   --# force off index_scan.   
set pg_strom.enable_gpusort to on;                                                                                                                                       
set client_min_messages to warning;
select * from strom_zero_test order by smlint_x;
 id | key | smlint_x | integer_x | bigint_x | real_x | float_x | nume_x | smlsrl_x | serial_x | bigsrl_x 
----+-----+----------+-----------+----------+--------+---------+--------+----------+----------+----------
(0 rows)

select * from strom_zero_test order by integer_x;
 id | key | smlint_x | integer_x | bigint_x | real_x | float_x | nume_x | smlsrl_x | serial_x | bigsrl_x 
----+-----+----------+-----------+----------+--------+---------+--------+----------+----------+----------
(0 rows)

select * from strom_zero_test order by bigint_x;
 id | key | smlint_x | integer_x | bigint_x | real_x | float_x | nume_x | smlsrl_x | serial_x | bigsrl_x 
----+-----+----------+-----------+----------+--------+---------+--------+----------+----------+----------
(0 rows)

select * from strom_zero_test order by real_x;
 id | key | smlint_x | integer_x | bigint_x | real_x | float_x | nume_x | smlsrl_x | serial_x | bigsrl_x 
----+-----+----------+-----------+----------+--------+---------+--------+----------+----------+----------
(0 rows)

select * from strom_zero_test order by float_x;
 id | key | smlint_x | integer_x | bigint_x | real_x | float_x | nume_x | smlsrl_x | serial_x | bigsrl_x 
----+-----+----------+-----------+----------+--------+---------+--------+----------+----------+----------
(0 rows)

select * from strom_zero_test order by nume_x;
 id | key | smlint_x | integer_x | bigint_x | real_x | float_x | nume_x | smlsrl_x | serial_x | bigsrl_x 
----+-----+----------+-----------+----------+--------+---------+--------+----------+----------+----------
(0 rows)

